helm/cluster/templates/clusterapi/_helpers_files.tpl

```diff
@@ -37,7 +37,7 @@
 {{- end }}
 
 {{- define "cluster.internal.kubeadm.files.cgroupv1" }}
-{{- if $.Values.internal.advancedConfiguration.cgroupsv1 }}
+{{- if $.nodePool.config.cgroupsv1 }}
```

Comment:
Where is `$.nodePool` coming from here?

Reply:
```
{{- $_ := set $ "nodePool" (dict "name" $nodePoolName "config" $nodePoolConfig) }}
```

Reply:
If you start to follow the actual chain of function calls, templates rendering and so on, you could start forgetting who you are, but also [find it here](https://github.com/giantswarm/cluster/blob/main/helm/cluster/templates/clusterapi/workers/kubeadmconfig.yaml#L16).